bindata/scripts/kargs.sh

```diff
@@ -5,54 +5,84 @@ command=$1
 shift
 declare -a kargs=( "$@" )
 ret=0
-args=$(chroot /host/ cat /proc/cmdline)
 
-IS_OS_UBUNTU=true; [[ "$(chroot /host/ grep -i ubuntu /etc/os-release -c)" == "0" ]] && IS_OS_UBUNTU=false
+if [[ -d "/bindata/scripts" ]];then
+    chroot_path="/host/"
+else
+    chroot_path="/"
+fi
+
+args=$(chroot "$chroot_path" cat /proc/cmdline)
+
+IS_OS_UBUNTU=true; [[ "$(chroot "$chroot_path" grep -i ubuntu /etc/os-release -c)" == "0" ]] && IS_OS_UBUNTU=false
 
 # Kernel args configuration isn't supported for Ubuntu now, so we shouldn't do anything here
```

Comment:
Maybe remove the comment about not supporting ubuntu?

Reply:
done